okhttp-testing-support/src/main/kotlin/okhttp3/internal/concurrent/TaskFaker.kt

```diff
@@ -360,15 +360,17 @@ class TaskFaker : Closeable {
       timeout: Long,
       unit: TimeUnit,
     ): T? {
-      taskRunner.withLock {
+      return taskRunner.withLock {
```

Comment:
what’s with this change?

Reply:
Kotlin 2 didn't like the inline with return?